src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -592,6 +591,9 @@ export default class FilingHistoryList extends Mixins(
         item.isTypeStaff = true
         item.notationOrOrder = filing.data.order?.orderDetails // should not be falsy
         item.planOfArrangement = filing.data.order?.effectOfOrder ? 'Pursuant to a Plan of Arrangement' : ''
+        item.putBackOnOrAdminDissolution = this.isTypePutBackOn({ name: filing.name }) ||
+        this.isTypeAdministrativeDissolution({ name: filing.name,
```

Comment:
I'm not sure why the linter didn't catch this, but it should be indented (since it's a continuation of the previous line).